style.scss

```diff
@@ -105,44 +105,59 @@ header {
 main {
     flex-grow: 1;
 
-    .work {
+    .hero-section {
+        padding: 10px;
```

Comment:
Since you use 16px as a basic offset, please increase the padding value to 16px here